raft/src/main/java/org/apache/kafka/raft/LeaderState.java

```diff
@@ -50,8 +50,8 @@ protected LeaderState(
         this.highWatermark = Optional.empty();
 
         for (int voterId : voters) {
-            boolean hasEndorsedLeader = voterId == localId;
-            this.voterReplicaStates.put(voterId, new VoterState(voterId, hasEndorsedLeader));
+            boolean hasAcknowledgedLeader = voterId == localId;
```

Comment:
Could we add a brief comment in the class javadoc about what we mean by "acknowledged"? It would be helpful to mention specifically that the set of unacknowledged voters are targets for `BeginQuorumEpoch` requests from the leader.

Reply:
done